icu_base.py

```diff
@@ -88,7 +89,12 @@ def build(self):
             tools.replace_in_file(run_configure_icu_file, "-MDd", flags)
             tools.replace_in_file(run_configure_icu_file, "-MD", flags)
 
-        self._replace_pythonpath() # ICU 64.1
+            toolset = self.settings.get_safe("compiler.toolset") or ""
+            if "LLVM" in toolset:
```

Comment:
Might this be case sensitive?  How about "LLVM" in toolset.upper()

Reply:
Good idea. I'll get that in.

Reply:
@datalogics-robb Patched it in the commit the work was done in.